latex/README.md

```
@@ -0,0 +1,29 @@
+## LaTeX generation of the RFCs from the .md files
+
+
+For the VS Code, it's good to use plugin https://marketplace.visualstudio.com/items?itemName=James-Yu.latex-workshop.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)